<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroundControlPointsView</class>
 <widget class="QWidget" name="GroundControlPointsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="pointsList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>75</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="northingLabel">
       <property name="text">
        <string>Latitude</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="northing">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>9</number>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="eastingLabel">
       <property name="text">
        <string>Longitude</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="easting">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>9</number>
       </property>
       <property name="minimum">
        <double>-360.000000000000000</double>
       </property>
       <property name="maximum">
        <double>360.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="elevationLabel">
       <property name="text">
        <string>Elevation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="elevation">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>20000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QToolBar" name="toolBar">
     <addaction name="actionApplySimilarity"/>
     <addaction name="actionRevert"/>
     <addaction name="actionDelete"/>
    </widget>
   </item>
  </layout>
  <action name="actionCopyLocationLatLon">
   <property name="text">
    <string>Latitude, Longitude</string>
   </property>
  </action>
  <action name="actionCopyLocationLonLat">
   <property name="text">
    <string>Longitude, Latitude</string>
   </property>
  </action>
  <action name="actionCopyLocationLatLonElev">
   <property name="text">
    <string>Latitude, Longitude, Elevation</string>
   </property>
  </action>
  <action name="actionCopyLocationLonLatElev">
   <property name="text">
    <string>Longitude, Latitude, Elevation</string>
   </property>
  </action>
  <action name="actionRevert">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/16x16/reset</normaloff>:/icons/16x16/reset</iconset>
   </property>
   <property name="text">
    <string>&amp;Revert Changes</string>
   </property>
   <property name="iconText">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Revert user changes to the active point</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/16x16/delete</normaloff>:/icons/16x16/delete</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete Point</string>
   </property>
   <property name="iconText">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete the active point</string>
   </property>
  </action>
  <action name="actionApplySimilarity">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/16x16/apply</normaloff>:/icons/16x16/apply</iconset>
   </property>
   <property name="text">
    <string>&amp;Apply Similarity Transform</string>
   </property>
   <property name="iconText">
    <string>Revert</string>
   </property>
   <property name="toolTip">
    <string>Apply Similarity Transform to correct geo-locations</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
